src/components/button-minor/button-minor-test.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import ButtonMinor from "./button-minor.component";
+import { ICONS } from "../icon/icon-config";
+import {
+  BUTTON_SIZES,
+  BUTTON_VARIANTS,
+  BUTTON_ICON_POSITIONS,
+} from "../button/button.config";
+
+export default {
+  title: "Button Minor/Test",
+  includeStories: "DefaultStory",
+  parameters: {
+    info: { disable: true },
+    chromatic: { disable: true },
+  },
+};
+
+const commonArgTypesButtonMinor = {
+  size: {
+    options: BUTTON_SIZES,
+    control: { type: "select" },
+  },
+  buttonType: {
+    options: BUTTON_VARIANTS,
+    control: {
+      type: "select",
+    },
+  },
+  iconType: {
+    options: [...ICONS, ""],
+    control: {
+      type: "select",
+    },
+  },
+  iconPosition: {
+    options: BUTTON_ICON_POSITIONS,
+    control: { type: "select" },
+  },
+  href: {
+    control: {
+      type: "text",
+    },
+  },
+};
+
+const commonArgsButtonMinor = {
+  size: "medium",
+  subtext: "",
+  buttonType: "secondary",
+  fullWidth: false,
+  disabled: false,
+  destructive: false,
+  noWrap: false,
+  href: undefined,
+  iconPosition: "before",
+};
+
+export const DefaultStory = ({ ...args }) => (
+  <ButtonMinor onClick={action("click")} {...args}>
+    Example Button
+  </ButtonMinor>
+);
+
+export const Default = ({ ...args }) => <ButtonMinor {...args} />;
+
+export const ButtonMinorDifferentTypes = ({ ...props }) => {
```

Comment:
```suggestion
export const ButtonMinorDifferentTypes = (props) => {
```

The `props` argument should be typed correctly.

Reply:
When I change that I get this error "Parameter 'props' implicitly has an 'any' type." Is that correct?

Reply:
Is this change acceptable @mkrds ?

`export const ButtonMinorDifferentTypes = ((props: JSX.IntrinsicAttributes & ButtonMinorProps) => {`

Reply:
I've edited my comment and suggestion :)

Reply:
Updated.